mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -1,44 +1,7 @@
 CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
-	FIELD field_11137 BLOCK_ENTITY Lnet/minecraft/class_2378;
-	FIELD field_11138 FEATURE Lnet/minecraft/class_2378;
 	FIELD field_11139 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_11140 DEFAULT_ENTRIES Ljava/util/Map;
-	FIELD field_11141 PARTICLE_TYPE Lnet/minecraft/class_2378;
-	FIELD field_11142 ITEM Lnet/minecraft/class_2348;
-	FIELD field_11143 POTION Lnet/minecraft/class_2348;
 	FIELD field_11144 REGISTRIES Lnet/minecraft/class_2385;
-	FIELD field_11145 ENTITY_TYPE Lnet/minecraft/class_2348;
-	FIELD field_11146 BLOCK Lnet/minecraft/class_2348;
-	FIELD field_11147 SURFACE_BUILDER Lnet/minecraft/class_2378;
-	FIELD field_11148 DECORATOR Lnet/minecraft/class_2378;
-	FIELD field_11149 CHUNK_GENERATOR_TYPE Lnet/minecraft/class_2378;
-	FIELD field_11150 MOTIVE Lnet/minecraft/class_2348;
-	FIELD field_11151 BIOME_SOURCE_TYPE Lnet/minecraft/class_2378;
-	FIELD field_11152 STAT_TYPE Lnet/minecraft/class_2378;
-	FIELD field_11153 BIOME Lnet/minecraft/class_2378;
-	FIELD field_11154 FLUID Lnet/minecraft/class_2348;
-	FIELD field_11155 DIMENSION Lnet/minecraft/class_2378;
-	FIELD field_11156 SOUND_EVENT Lnet/minecraft/class_2378;
-	FIELD field_11157 CARVER Lnet/minecraft/class_2378;
-	FIELD field_11158 CUSTOM_STAT Lnet/minecraft/class_2378;
-	FIELD field_11159 STATUS_EFFECT Lnet/minecraft/class_2378;
-	FIELD field_11160 ENCHANTMENT Lnet/minecraft/class_2378;
-	FIELD field_16643 CHUNK_STATUS Lnet/minecraft/class_2348;
-	FIELD field_16644 STRUCTURE_FEATURE Lnet/minecraft/class_2378;
-	FIELD field_16645 STRUCTURE_PIECE Lnet/minecraft/class_2378;
-	FIELD field_16792 RULE_TEST Lnet/minecraft/class_2378;
-	FIELD field_16793 STRUCTURE_POOL_ELEMENT Lnet/minecraft/class_2378;
-	FIELD field_16794 STRUCTURE_PROCESSOR Lnet/minecraft/class_2378;
-	FIELD field_17166 VILLAGER_TYPE Lnet/minecraft/class_2348;
-	FIELD field_17167 VILLAGER_PROFESSION Lnet/minecraft/class_2348;
-	FIELD field_17429 CONTAINER Lnet/minecraft/class_2378;
```

Comment:
What is this one now called?

Reply:
IIRC this is `MENU`, and there was a third name change as well. I'll look it up in Enigma.

Reply:
Yes, thats what I was worried about. For now keep it as container.

Reply:
The full list of changed names is:
- `BLOCK_ENTITY` => `BLOCK_ENTITY_TYPE`
- `DIMENSION` => `DIMENSION_TYPE`
- `CONTAINER` => `MENU`
- `STATUS_EFFECT` => `MOB_EFFECT`

I don't think we should give other names to registered objects, as we don't rename blocks/items/entities either (there are also awful names like `area_effect_cloud` there).

If we do rename some registries, I think `CONTAINER` and `STATUS_EFFECT` can have their old names and the two others can have `_TYPE` at the end.

Reply:
I'd rather keep `STATUS_EFFECT`. Besides, are there that many registries that it's essential to auto-generate names for them ?

Reply:
There are 42 registries. Also, I don't see any point in mapping things that can be mapped automatically.

Reply:
Well, consistency is the main issue. I think we should override generated names when they conflict with the established ones.

Reply:
I added back the old names of `CONTAINER` and `STATUS_EFFECT`. (renaming those can be a discussion issue, anyway)